src/lvl_script_commands.c

```diff
@@ -1556,7 +1597,6 @@ static void new_object_type_check(const struct ScriptLine* scline)
     struct ObjectConfigStats* objst = get_object_model_stats(tmodel);
     memset(objst->code_name, 0, COMMAND_WORD_LEN);
     snprintf(objst->code_name, COMMAND_WORD_LEN, "%s", scline->tp[0]);
-    objst->name_stridx = 201;
```

Comment:
Is this removed on purpose? Note that 201 is the 'empty' string.

Reply:
It's an unused field, do you plan on using it in the future? Otherwise field can be scrapped